src/libraries/Microsoft.PowerFx.Core/Functions/Delegation/DelegationStrategies/DelegationValidationStrategy.cs

```diff
@@ -400,6 +400,11 @@ protected bool IsValidCallNodeInternal(CallNode node, TexlBinding binding, Opera
             }
 
             var callInfo = binding.GetInfo(node);
+            if (binding.DelegationHintProvider.TryGetWarning(node, callInfo.Function, out var warning))
```

Comment:
>DelegationHintProvider [](http://example.com/codeflow?start=24&length=22)

This can be null, so need to use something like `if(binding.DelegationHintProvider?.TryGetWarning(...) == true)`